deploy/28_factory_deployed_contracts.ts

```diff
@@ -65,9 +65,50 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
       });
       hre.deployments.log('Deployed a "CxipERC721Proxy" empty contract for block explorer verification purposes.');
     }
+
+    const holographUtilityToken: Contract | null = await hre.ethers.getContractOrNull(
```

Comment:
This is just so the HLG token contract shows on block explorers?

Reply:
Yes, the contract is a dummy and will not function as an actual ERC20 contract but rather just revert

Reply:
where does it revert?

Reply:
It does not revert. We use getContractOrNull purposefully. And then we just check for null.